zcash_primitives/src/jubjub/mod.rs

```diff
@@ -227,7 +227,7 @@ impl JubjubBls12 {
             let mut pedersen_hash_generators = vec![];
 
             for m in 0..6 {
-                use byteorder::{LittleEndian, WriteBytesExt};
+                use byteorder::{WriteBytesExt, LittleEndian}; //TODO: swapped between branches??
```

Comment:
Can you explicate further?

Reply:
for some reason the order was swapped between the uda and master branches. Probably no reason, so I went with the master